ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -1098,6 +1125,7 @@ static PrjFS_Result HandleFileNotification(
     PrjFS_Result result = s_callbacks.NotifyOperation(
         0 /* commandId */,
         relativePath,
```

Comment:
What is preventing us from sending a null `relativePath` path to the provider?  What will `relativePath` be when a hardlink is created outside of the repo to a file that's inside the repo? 

(As discussed in a separate comment these checks could also be moved into `OnNotifyOperation`)

Reply:
I've updated to send "" for relative paths in this case.  If GetRelativePath gets "", it will return "".